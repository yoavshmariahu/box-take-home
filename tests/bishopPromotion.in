k a1
K e5
b b4
B d2

[G S R P]
[p r s g]

move b4 c5 promote
move d2 c1 promote
