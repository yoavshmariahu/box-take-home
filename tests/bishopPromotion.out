UPPER player action: move d2 c1 promote
5 |__|__|+b|__| K|
4 |__|__|__|__|__|
3 |__|__|__|__|__|
2 |__|__|__|__|__|
1 | k|__|+B|__|__|
    a  b  c  d  e

Captures UPPER: G S R P
Captures lower: p r s g

lower> 
